### ISSUE TITLE
Postgres error: invalid input syntax for integer: "" when run "deepdive do sentences"

### ISSUE THREAD

**Issue by fingertap** [HazyResearch/deepdive#577]
Hey there! I have encoutered a problem when trying to use [the whole article dataset](http://research.signalmedia.co/newsir16/signal-dataset.html) rather than the provided "articles-100*.tsv" in the spouse example. I download the raw articles in json format and parse it with python to extract `id` and `content` of the articles and saved them as tsv format. Loading articles with `deepdive do articles` is okay but when I run `deepdive do sentences`, things go wrong.
The log is:

``` log
2016-08-15 02:37:57.529585 ERROR:  invalid input syntax for integer: ""
2016-08-15 02:37:57.529644 CONTEXT:  COPY dd_tmp_sentences, line 63020, column doc_offsets: "{""}"   
2016-08-15 02:37:57.575243 [ERROR] command='"$DEEPDIVE_APP"/udf/nlp_markup.sh': PID 6394: finished with non-zero exit status (0)
2016-08-15 02:37:57.575530 /root/local/util/compute-driver/local/compute-execute: line 138:  6399 Terminated              mkmimo process-*.output \> ou
tput_computed-*
2016-08-15 02:37:57.575625 /root/local/util/compute-driver/local/compute-execute: line 138:  6400 Terminated              deepdive-load "$output_relati
on_tmp" output_computed-*
```

It indicates there are some child process failed with error "invalid input syntax for integer". It seems that empty string had been passed and caused a Postgresql error, which I do not have access to fix it. Any ideas?

Edit:
If I rerun the command `deepdive do sentences`, there may be no error message shown in terminal except

``` log
‘run/ABORTED’ -> ‘20160815/030924.343102844’
```

In `/var/log/postgresql/postgres*.log` the error message gives

``` log
2016-08-15 03:18:22 UTC ERROR:  invalid input syntax for integer: ""
2016-08-15 03:18:22 UTC CONTEXT:  COPY dd_tmp_sentences, line 62171, column doc_offsets: "{""}"
2016-08-15 03:18:22 UTC STATEMENT:  COPY  dd_tmp_sentences FROM STDIN
2016-08-15 03:44:09 UTC LOG:  could not send data to client: Connection reset by peer
2016-08-15 03:44:09 UTC STATEMENT:  COPY ( SELECT R0.id AS "articles.R0.id", R0.content AS "articles.R0.content"
        FROM articles R0

                  ) TO STDOUT
2016-08-15 03:44:09 UTC FATAL:  connection to client lost
2016-08-15 03:44:09 UTC STATEMENT:  COPY ( SELECT R0.id AS "articles.R0.id", R0.content AS "articles.R0.content"
        FROM articles R0

                  ) TO STDOUT
```

**Comment by DarkDataAnalytics** [comment]
I have the same issue. I am running an EC2 instance with Ubuntu 16.04.

**Comment by DarkDataAnalytics** [comment]
I tried to run the same thing on an EC2 instance with Ubuntu 14.04 and now I have issues with JQ ( tsk is not a valid format ).

**Comment by netj** [comment]
This seems to be a bug in Bazaar/Parser producing malformed values.  We'll soon release a more direct way to use CoreNLP which won't suffer from these nasty issues.  The parsed 1M articles using it is already available for download from #566.
